src/sentry/autofix/utils.py

```diff
@@ -32,12 +32,26 @@ class AutofixStatus(str, enum.Enum):
     WAITING_FOR_USER_RESPONSE = "WAITING_FOR_USER_RESPONSE"
 
 
+class FileChange(BaseModel):
+    path: str
+    content: str | None = None
+    is_deleted: bool = False
+
+
+class CodebaseState(BaseModel):
+    repo_external_id: str | None = None
+    file_changes: list[FileChange] = []
+    is_readable: bool = False
+    is_writeable: bool = False
+
+
```

Comment:
why do we need file changes passed down here?

Reply:
we don't but this was just copying the base models over